package.json

```diff
@@ -3,45 +3,44 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "npm run serve",
+    "start": "node server.js",
     "watch": "npm run watch:server & npm run watch:client",
     "watch:server": "nodemon server.js",
     "watch:client": "webpack -w --progress --colors",
     "dev": "webpack-dev-server",
     "build": "webpack -p",
-    "serve": "npm run build && node server.js",
     "lint": "eslint src || exit 0",
     "test": "jest --env=jsdom"
   },
   "dependencies": {
-    "axios": "^0.15.3",
-    "basscss-sass": "^4.0.0",
-    "normalize.css": "^5.0.0",
-    "react": "^15.4.1",
-    "react-dom": "^15.4.1"
+    "basic-auth-connect": "^1.0.0",
+    "cfenv": "^1.0.3",
+    "express": "^4.14.0"
   },
   "devDependencies": {
+    "axios": "^0.15.3",
     "babel-cli": "^6.18.0",
     "babel-core": "^6.18.2",
     "babel-eslint": "^7.1.1",
     "babel-jest": "^17.0.2",
     "babel-loader": "^6.2.8",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
-    "basic-auth-connect": "^1.0.0",
-    "cfenv": "^1.0.3",
+    "basscss-sass": "^4.0.0",
     "css-loader": "^0.26.1",
     "eslint": "^3.11.1",
     "eslint-config-airbnb": "^13.0.0",
     "eslint-plugin-babel": "^4.0.0",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.8.0",
-    "express": "^4.14.0",
     "extract-text-webpack-plugin": "^1.0.1",
     "jest": "^17.0.3",
+    "react": "^15.4.1",
```

Comment:
what's the thinking behind moving these (react, reactdom, basscss, etc) into devdependencies? i always thought of devdependencies as for development-related scripts (e.g. unit testing, packaging, compiling) vs. dependencies for packages that are used for production... fwiw, when you eject a `create-react-app` app (https://github.com/facebookincubator/create-react-app), they put react and reactdom into the dependencies bucket